package.json

```diff
@@ -17,7 +17,7 @@
     "build": "true"
   },
   "dependencies": {
-    "base32-crockford-browser": "^1.0.0",
+    "@18f/identity-personal-key-input": "file:./app/javascript/packages/personal-key-input",
```

Comment:
This turned out later to be a problem with the `es5-safe` package (see #4298). It should be fine to leave it out altogether.

```suggestion
```

You'll also need to rerun `yarn install` to pick up the `yarn.lock` revisions.